roles/ceph-rbd-mirror/tasks/docker/copy_configs.yml

```diff
@@ -23,7 +23,21 @@
   register: statconfig
   when: "item | length > 0"
 
-- name: try to fetch ceph config and keys
+- name: try to fetch ceph keys
+  copy:
+    src: "{{ fetch_directory }}/{{ fsid }}/{{ item.0 }}"
+    dest: "{{ item.0 }}"
+    owner: "{{ ceph_uid }}"
+    mode: 0600
```

Comment:
You can do this in one task:
mode: "{{ (item.0 | search("keyring")) | ternary('0600', '0644') }}"

Reply:
@colonwq could you please address @andymcc's comment ?